src/jdk.crypto.ec/share/native/libsunec/impl/mpi.c

```diff
@@ -37,6 +37,12 @@
  * Last Modified Date from the Original Code: Nov 2019
  *********************************************************************** */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2018, 2022 All Rights Reserved
```

Comment:
This should say "2023, 2023", not "2018, 2022".

Reply:
Fixed it. The latest testing looks good.